docs/flow-results-connector.md

```diff
@@ -5,7 +5,8 @@ This connector is designed in Apache NiFi to automate the process of transferrin
 <br />
 
 1. __Flow results API endpoint__
-The schema for the table is derived from the descriptor of a package retrieved from `/flow-results/packages/[form_uuid]`. Once the schema for the table is generated, a table is created in the database. Data is then retrieved from the response endpoint: `/flow-results/packages/[form_uuid]/responses`. The connector requires a JSON configuration with the keys `form_uuid`, `base_url` and optionally, the `table_name` the user desires the data to be stored in. The JSON config can is formatted as below:
+
+The schema for the table is derived from the descriptor of a package retrieved from `/flow-results/packages/[form_uuid]`. Once the schema for the table is generated, a table is created in the database. Data is then retrieved from the response endpoint: `/flow-results/packages/[form_uuid]/responses`. The connector requires a JSON configuration in "API configuration for flow results" processor within "API Batch pulls" process group with the keys `form_uuid`, `base_url` and optionally, the `table_name` the user desires the data to be stored in. The JSON config can is formatted as below:
 
```

Comment:
```markdown
We derive the schema for the table from the package descriptor at `flow-results/packages/[form_uuid]` and then create the table in the PostgreSQL database. Data is then retrieved from the response endpoint: `/flow-results/packages/[form_uuid]/responses`. The connector requires a JSON configuration in "API configuration for flow results" processor within "API Batch pulls" process group with the keys `form_uuid`, `1base_url` and optionally, the `table_name` the user desires to store the data. The JSON config is as below:
```

Reply:
How does this sound:
1. __Batch pulls from the API__

The API endpoint `/flow-results/packages/[form_uuid]` returns the package descriptor that is used to derive the PostgreSQL table schema. Data is then retrieved from the response endpoint `/flow-results/packages/[form_uuid]/responses` and inserted to the database. The connector requires a JSON configuration in "API configuration for flow results" processor within the "API Batch pulls" process group with the keys `form_uuid`, `base_url` and optionally, the `table_name` the user desires the data to be stored in. The JSON config can is formatted as below:

Reply:
> The JSON config can be formatted as below

I think this should be "The JSON config format is as below:"

Otherwise sounds good.